### PR TITLE
[cloud-provider-vsphere] Narrower permissions

### PR DESCRIPTION
**Pull request by zuzzas**
## Description

Provide internal documentation after testing Deckhouse on vSphere with narrower permission set.

## Why do we need it, and what problem does it solve?

Current permissions set is too wide.

## What is the expected result?

An attempt was made to limit permission set.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-vsphere
type: chore
summary: Provide internal documentation after testing Deckhouse on vSphere with narrower permission set.
impact_level: low
```

### CODE REVIEW

File: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT_RU.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ govc tags.attach -c k8s-zone k8s-zone-x1-a /X1/datastore/x1_lun_1
 ### Права
 
 > Ввиду разнообразия подключаемых к vSphere SSO-провайдеров шаги по созданию пользователя в данной статье не рассматриваются.
+<!-- markdownlint-disable-next-line MD028 -->
+> Эта роль включает в себя все возможные права для всех компонентов Deckhouse.
+> При необходимости получения более гранулярных прав обратитесь в тех. поддержку Флант.
```
Comment (apolovov) [inline ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT_RU.md:75]:
```suggestion
> При необходимости получения более гранулярных прав обратитесь в тех. поддержку Фланта.
```

File: docs/site/_includes/getting_started/vsphere/STEP_ENV_RU.md [hunk 2]
```diff
@@ -68,6 +68,9 @@ govc tags.attach -c k8s-zone test_zone /DC/datastore/test_lun
 ### Права
 
 > Ввиду разнообразия подключаемых к vSphere SSO-провайдеров, шаги по созданию пользователя в данной статье не рассматриваются.
+<!-- markdownlint-disable-next-line MD028 -->
+> Эта роль включает в себя все возможные права для всех компонентов Deckhouse.
+> При необходимости получения более гранулярных прав обратитесь в тех. поддержку Флант.
```
Comment (apolovov) [inline docs/site/_includes/getting_started/vsphere/STEP_ENV_RU.md:73]:
```suggestion
> При необходимости получения более гранулярных прав обратитесь в тех. поддержку Фланта.
```

File: docs/site/_includes/getting_started/vsphere/STEP_ENV.md [hunk 3]
```diff
@@ -68,6 +68,9 @@ StorageClasses will be created automatically for each Datastore that is tagged w
 ### Permissions
 
 > We've intentionally skipped User creation since there are many ways to authenticate a user in the vSphere.
+>
+> This all-encompassing Role should be enough for all Deckhouse components.
+> If you require a more granular Role, please, contact Flant support.
```
Comment (z9r5) [inline docs/site/_includes/getting_started/vsphere/STEP_ENV.md:73]:
```suggestion
> If you require a more granular Role, please, contact the Deckhouse community.
```

Comment (z9r5) [reply docs/site/_includes/getting_started/vsphere/STEP_ENV.md:73]: @zuzzas @apolovov For discussion...

File: docs/site/_includes/getting_started/vsphere/STEP_ENV_RU.md [hunk 4]
```diff
@@ -68,6 +68,9 @@ govc tags.attach -c k8s-zone test_zone /DC/datastore/test_lun
 ### Права
 
 > Ввиду разнообразия подключаемых к vSphere SSO-провайдеров, шаги по созданию пользователя в данной статье не рассматриваются.
+>
+> Эта роль включает в себя все возможные права для всех компонентов Deckhouse.
+> При необходимости получения более гранулярных прав обратитесь в тех. поддержку Фланта.
```
Comment (z9r5) [inline docs/site/_includes/getting_started/vsphere/STEP_ENV_RU.md:73]:
```suggestion
> При необходимости получения более гранулярных прав обратитесь к [сообществу Deckhouse](https://deckhouse.io/community/about.html).
```

File: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md [hunk 5]
```diff
@@ -70,6 +70,9 @@ StorageClasses will be created automatically for each Datastore that is tagged w
 ### Permissions
 
 > We've intentionally skipped User creation since there are many ways to authenticate a user in the vSphere.
+>
+> This all-encompassing Role should be enough for all Deckhouse components.
+> If you require a more granular Role, please, contact Flant support.
```
Comment (z9r5) [inline ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md:75]:
```suggestion
> If you require a more granular Role, please, contact [the Deckhouse community](https://deckhouse.io/community/about.html).
```

**Review by z9r5 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Pull request merged**